.murdock

```diff
@@ -131,6 +136,22 @@ print_worker() {
         echo "-- running on worker ${DWQ_WORKER} thread ${DWQ_WORKER_THREAD}, build number $DWQ_WORKER_BUILDNUM."
 }
 
+test_hash_calc() {
+    local appdir=$1
+    local bindir=$2
+
+    md5sum $appdir/tests/* $bindir/*.elf | md5sum | cut -f1 -d' '
+}
+
+test_cache_get() {
+    test "${ENABLE_TEST_CACHE}" = "1" || return 1
+    test -n "$(redis-cli get $1)" > /dev/null
```

Comment:
Any reason why Redis is added as a dependency here, while previously we just used the FS as a "database"?

Reply:
When I wrote this comment, I didn't know Redis can also be setup as a LRU cache (this should also be mentioned in the doc I asked for in https://github.com/RIOT-OS/RIOT/pull/10576#discussion_r240048996 ;-)). I only knew it as key-value store.